frontend/public/components/_global-notification.scss

```diff
@@ -13,7 +13,7 @@ $global-notification-text: #fff;
 }
 
 .co-global-notification + .co-global-notification {
-  border-top: 1px solid #00659c;
+  box-shadow: inset 0 10px 10px -10px rgba(0,34,53,0.7);
```

Comment:
Not new to this PR, but I recommend using nesting to keep this rule associated with the one above.  So on line 13 do:

```

+  .co-global-notification {
  box-shadow: inset 0 10px 10px -10px rgba(0,34,53,0.7);
}
```